.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,81 @@
+name: TileDB-Spark
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'master'
+  pull_request:
+
+jobs:
+  Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-10.15 ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: unset SYSTEM; ./gradlew checkFormat;
+        shell: bash
+        name: checkFormat
+      - run: unset SYSTEM; ./gradlew assemble;
+        shell: bash
+        name: assemble
+      - run: unset SYSTEM; ./gradlew test;
+        name: test
+        shell: bash
+
+  Release:
+    if: startsWith(github.ref, 'refs/tags/')
+    needs: [Test]
+    name: Create-Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create-Jars
+        run: unset SYSTEM; set +e; ./gradlew assemble; ./gradlew shadowJar; mkdir jars/; cp ./build/libs/*.jar jars/
+
+      - name: Create Release
+        id: create_release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          files: jars/*.jar
+          tag_name: ${{ github.event.release.tag_name }}
+          name: ${{ github.event.release.tag_name }}
+          body: ${{steps.github_release.outputs.changelog}}
+          draft: false
+          prerelease: false
+
+  Maven:
+    needs: [ Release ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
```

Comment:
Should we have a `setup-java` step above as well to make sure we don't end up with mismatched JVM versions?

Reply:
Done!